wordpress-mobile/AztecEditor-iOS
# Fix copy method to keep other text data when adding custom attributedstring data.

Fixes #

This PR adds extra checks to the copy code to make sure we are keeping the other data in the pasteboard 

To test:
 - Start the demo app
 - Copy some content from it
 - Paste it on the Reminders or Notes app
 - Check that the paste worked.
